src/glass/package.json

```diff
@@ -52,10 +53,12 @@
     "@mdi/angular-material": "5.9.55",
     "@ngx-translate/core": "^13.0.0",
     "@swimlane/ngx-charts": "^17.0.1",
+    "cypress": "^7.1.0",
```

Comment:
is this the right place for it? Maybe the dev dependency is better?

Reply:
Ack, I'll move it to `devDependencies` :+1: